src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -793,7 +793,16 @@ public void UpdateReportTabStatistics(Patch patch)
                 var dataset = speciesPopulationChart.GetDataSet(entry.Key.FormattedName);
 
                 var extinctInPatch = entry.Value <= 0;
-                var extinctEverywhere = snapshot.RecordedSpeciesInfo[entry.Key].Population <= 0;
+                var extinctEverywhere = false;
+
+                // We test if the species info was recorded before using it.
+                // This is especially for compatibility with older versions, to avoid crashed due to an invalid key.
+                // TODO: Use a proper save upgrade (e.g. summing population to generate info).
+                SpeciesInfo speciesInfo;
```

Comment:
This can be inlined into the next line

Reply:
I didn't know about it. Corrected (I think).